packages/examples/src/common/model.ts

```diff
@@ -0,0 +1,34 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) 2018-2022 TypeFox GmbH (http://www.typefox.io). All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+import * as cp from 'child_process';
+import { languages } from 'monaco-editor';
+import { ServerOptions } from 'ws';
+
+export enum LanguageCli {
+    /** https://nodejs.org/api/cli.html  */
+    node = 'node',
+    /** https://docs.oracle.com/en/java/javase/21/docs/specs/man/java.html */
+    java = 'java'
+}
+export interface LanguageServerRunConfig {
+    serverName: string;
+    pathName: string;
+    serverPort: number;
+    runCommand: LanguageCli;
```

Comment:
What about
```js
runCommand: LanguageCli | string;
```
This way you don't overlimit the potential value, but can still pin-point what is fixed via the enum.